frontend/website/pages/docs/getting-started.mdx

```diff
@@ -17,10 +17,10 @@ Welcome to Coda Genesis Program applicants! This section will walk you through t
 
 Note: Windows is not officially supported at this time. However, community members succeeded in setting up nodes using Windows Subsystem for Linux. Click [here](https://codawiki.com/index.php?title=Coda_on_Windows) for instructions on using Windows, which are created by the community.
 
-**Hardware**: Sending and receiving coda does not require any special hardware, but running a block producer on the Coda network currently requires:
+**Hardware**: Sending and receiving coda does not require any special hardware, but the recommended requirements for running a block producer on the Coda network is currently:
```

Comment:
@figitaki  does it make sense to add this note?  "Please note: the increase in hardware requirements is a temporary measure for this release only, and is not reflective of the hardware requirements for future testnets or the mainnet."